### PR TITLE
Fix(eos_designs): Minor issues in new PTP feature

### LINKED ISSUE #2328: Bug: PTP feature in eos_designs has flaws
### Issue Summary

Vu La has found the following:

1. source_ip didn't generate any configuration
2. smpte profile configured an announce internal of -1 instead of the documented -2.
3. Adding a second ptp only link yielded no ptp intervals, only a description, no switchport, mtu, and ptp enable

### Which component(s) of AVD impacted

eos_designs

### How do you run AVD ?

Ansible CLI (with virtual-env or native python)

### Steps to reproduce

```shell
1) Define source_ip under a node. This will not generate the relevant config.
2) Wrong defaults defined for SMPTE2059-2
3) Only first core interface defined generates the proper config - not the 2nd one.
```

### Relevant log output

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by nielsjlarsen**
## Change Summary

Fixes bugs found (by Vu La - thanks!) in the PTP feature in eos_designs.

## Related Issue(s)

Fixes #2328 

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes

1) Fixed typo in PTP profile "smpte2059-2":
    announce interval changed from "-1" to "-2"

2) Fixed typo in eos_design_facts.py:

Old: "source": {"ip": get(self._switch_data_combined, "ptp.source.ip")},
New: "source": {"ip": get(self._switch_data_combined, "ptp.source_ip")},

3) Adding a second ptp only link (using core_interfaces) yielded no ptp intervals, only a description, no switchport, mtu, and ptp enable.
This is fixed in https://github.com/aristanetworks/ansible-avd/pull/2322 and https://github.com/aristanetworks/ansible-avd/pull/2327

## How to test

molecule converge --scenario-name eos_designs_unit_tests -- 

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by carlbuchmann — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml [hunk 1]
```diff
@@ -60,6 +60,5 @@ l3leaf:
               delay_resp: 12
               follow_up: 13
               sync: 14
-        clock_identity_prefix: "22:33:44"
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml:63]: why are we removing the test for a custom prefix?

**Review by emilarista — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml (hunk 1 above)
Comment (nielsjlarsen) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml:63]: There's plenty of tests of custom prefixes on the other nodes in the test repo. By removing this we're effectively testing the fully auto-generated ClockID, which unfortunately wasn't tested earlier.

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml:63]: ack. Thanks

**Pull request merged**